content/docs/community/blog-posts.md

```diff
@@ -14,8 +14,6 @@ See also [Related Tools](/docs/tools/related-tools).
 |:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------|
 | **Post**                                                                                                                                                                                                                                                               | **Description**                                                                                                                                       | **Posting Date**    |
 | [What is BDD?](http://www.thinkcode.se/blog/2019/04/24/what-is-bdd)                                                                                                                                                                                                    | A definition of BDD by Thomas Sundberg                                                                                                               | April 24, 2019      |
-| [Getting Started With Cucumber Pro](https://jayson.codes/and/blogs/2018/08/getting-started-with-cucumber-pro/)                                                                                                                                                         | Getting up and running with Cucumber Pro and some handy tips by Jayson Smith                                                                          | August 9, 2018      |
-| [Closing A Two Year Old Cucumber Ruby Issue](https://jayson.codes/and/blogs/2018/08/cucumber-closing-1021/)                                                                                                                                                            | Jayson Smith goes over closing a two year old issue in the Ruby Cucumber project around code style                                                    | August 5, 2018      |
```

Comment:
I did rather like this blogpost. Could we get a working link from Jayson? (the other blogpost about Cucumber pro might not be relevant anymore)